### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S po,interfaces -L
ciph,fo,ro,nd,parm,parms,readd,reenabled,msdos,sav,tage,datas`

### CODE REVIEW

**Comment by thezoggy** [comment]
We can't change skin text without invalidating translations, see previous codespell pull

**Comment by Safihre** [comment]
Indeed, corrections to English texts in `skintext` need to be done through `en.po` otherwise all translations need to be manually corrected in Transifex.

File: sabnzbd/skintext.py [hunk 1]
```diff
@@ -52,12 +52,12 @@
     "sch-rss_scan": TT("Read RSS feeds"),  #:  #: Config->Scheduler
     "sch-remove_failed": TT("Remove failed jobs"),  #: Config->Scheduler
     "sch-remove_completed": TT("Remove completed jobs"),  #: Config->Scheduler
-    "sch-pause_all_low": TT("Pause low prioirty jobs"),  #: Config->Scheduler
-    "sch-pause_all_normal": TT("Pause normal prioirty jobs"),  #: Config->Scheduler
-    "sch-pause_all_high": TT("Pause high prioirty jobs"),  #: Config->Scheduler
-    "sch-resume_all_low": TT("Resume low prioirty jobs"),  #: Config->Scheduler
-    "sch-resume_all_normal": TT("Resume normal prioirty jobs"),  #: Config->Scheduler
-    "sch-resume_all_high": TT("Resume high prioirty jobs"),  #: Config->Scheduler
+    "sch-pause_all_low": TT("Pause low priority jobs"),  #: Config->Scheduler
```
Comment (Safihre) [inline sabnzbd/skintext.py:55]: All these are already in en.po, so if you remove these I can merge.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**